tests/lib/cli.ts

```diff
@@ -237,3 +251,33 @@ test.serial('Event scann::end should write a message in the spinner', async (t)
 
     (Sonar.prototype.executeOn as sinon.SinonStub).restore();
 });
+
+test.serial('User should be notified if there is a new version of sonar', async (t) => {
+    const newUpdate = {
+        current: '0.2.0',
+        latest: '0.3.0',
+        name: '@sonarwhal/sonar',
+        type: 'minor'
+    };
+
+    const expectedMessage = `Update available ${chalk.red(newUpdate.current)}${chalk.reset(' → ')}${chalk.green(newUpdate.latest)}
+Please update to the newest version ${chalk.green(newUpdate.latest)}
+See ${chalk.cyan('https://sonarwhal.com/about/changelog.html')} for details`;
+
+    t.context.notifier.update = newUpdate;
+
+    await cli.execute('http://localhost/');
+
+    t.true(t.context.notifier.notify.calledOnce);
+    t.is(t.context.notifier.notify.args[0][0].message, expectedMessage);
+});
+
+test.serial(`User shouldn't be notified if the current version is up to date`, async (t) => {
+    let undefinedUpdate;
+
+    t.context.notifier.update = undefinedUpdate;
```

Comment:
I think you should do directly `t.context.notifier.update = void 0` instead of use a variable.